lib/restrictedsession/restricted.go

```diff
@@ -298,6 +306,27 @@ func (l *auditEventLoop) close() {
 	l.wg.Wait()
 }
 
+// checkLsmBpf checks that eBPF is one of the enable
+// LSM "modules".
+func checkBpfLsm() error {
+	lsmInfo := "/sys/kernel/security/lsm"
```

Comment:
nit: can we change to constant?

Reply:
Yep. For some reason I thought that Go doesn't support `const` at local level :facepalm: